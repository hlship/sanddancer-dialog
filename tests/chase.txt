take jade
brood stories
(setup lighting)
(now) (#pursuit has completed)
(now) (#rainstorm has completed)
%% Prevent pursuit from triggering:
(now) (#burrow is visited)
e
n
n
n
x shadows
n
s
s
