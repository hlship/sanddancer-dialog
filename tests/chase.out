

Middle of Nowhere (in the pickup truck)
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air. On the dash you
can see a piece of jade. The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> take jade
You take the piece of jade.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> brood stories
There are dark spirits who roam the earth, little Knock. Grandma used to say
that, holding you tight and stroking your hair. There are dark spirits who roam
the earth, but you're not alone. Oh, no. I'm here. (She'd kiss your head and
you'd squeeze her back.) But others are watching out for you too. You have three
animal guardians, hmm? Spirits who are always watching over you. Oh, you can't
always trust them to know what's best. Remember that, Knock. But when you need
help, they'll come, and protect you from the worser things in the world.

Mom would yell at Grandma a lot for filling your head with that new-age
bullshit. Grandma grew up white and midwestern and Baptist, but had started
wearing things with feathers and playing the pan flute by the time your were
born. She seemed to really like having a son-in-law who was Native American or
American Indian or Indigenous People or whatever she'd decided the term was that
week, and she was pretty pissed when Mom left him. Anyway. Her stories were
mostly BS, you guess, but some of them stuck with you. When it's dark you still
wonder if your spirit animals are out there, somewhere, and what the hell is
taking them so long to find you.

> (setup lighting)
Query succeeded: (setup lighting)
> (now) (#pursuit has completed)
> (now) (#rainstorm has completed)
> (now) (#burrow is visited)
> e
(first attempting to get out of the pickup truck)
You work the rusty truck door open. You get out of the pickup truck.

You stop in mid-stride as your eyes flick to the horizon. Way out in the desert,
silhouetted on a rise, a distant figure stands. As you watch he raises a hande
and waves.

You walk east for a long time, thoughts turning over like the sand under your
sneakers. You think about your grandma's stories. You think about Ocean and how
pissed she's gonna be that you're not home. Like you tell her not to wait up for
you but she always does anyway.

Some wind kicks up and chucks sand in your face. You blink it out and kinda
realize while you're doing it that you haven't really been paying attention to
where you've been going. Another gust whines through the sagebrush and you
realize with a kind of heavy feeling that with all the dust this wind is kicking
up you can't see a damn thing. You can't see your pickup, you can't see the
electrical tower. Not even your damn footprints. You're lost. Well, shit.

Open Desert
The desert stretches around you in all directions, and you plod wearily through
the sand.

You can sort of see some clumps of sagebrush, a dry streambed, and a looming
cactus.

Something snarls off in the distance, a warning growl.

> n
You move nervously forward.

Open Desert
Clouds of dust swirl through the air, and you shiver in the chill night air.

Half-lost in the shadows, you see a dry streambed, a looming cactus, and some
clumps of sagebrush and that's about it.

A howl from behind you, and an answer from somewhere ahead. Coyotes. Hungry,
sounds like.

> n
You jog forward, eyes snapping left and right at the moving shadows around you.

Open Desert
Your tracks are swallowed up in moments by the billowing sand, and you plod
wearily through the sand.

Nearby: a stand of cactus, a low bluff, and a looming cactus. Nothing to write
home about.

Coyotes call all around you now, circling in the dark, unseen, but coming
closer, closer.

> n
You're running now, jumping over sagebrush and stumbling on rocks, as snarling
animal forms close in around you.

Open Desert
The desert stretches around you in all directions, and you shiver in the chill
night air.

You can sort of see a looming cactus, a dry streambed, and a stand of cactus.

You can smell them now, wild, feral. Shifting shadows whirl around you, panting,
whining with the thrill of the chase.

> x shadows
All you can do is run.

The pack's at your heels, snapping, growling, and you've never in your life been
this terrified.

> n
You run.

Open Desert
The dust storm rages around you, and god damn it's dark.

Nearby: some clumps of sagebrush, a stand of cactus, and a dry streambed.
Nothing to write home about.

Snarls and teeth and mangy fur on every side.

> s
You run.

Open Desert
Your tracks are swallowed up in moments by the billowing sand, and you plod
wearily through the sand.

Half-lost in the shadows, you see some dry bones, a looming cactus, and a stand
of cactus and that's about it.

Snarls and teeth and mangy fur on every side.

> s
You run.

Open Desert
The dust storm rages around you, and you shiver in the chill night air.

You can sort of see a dead cow, a dull grey boulder, and a looming cactus.

CHASE COMPLETE

> 
